src/components/Form/Form.js

```diff
@@ -33,7 +66,23 @@ const form = () => {
     const updatedFormElement = {
       ...quizForm[formState.step]['values'][inputIdentifier]
     };
-    updatedFormElement.value = event.target.value;
+
+    if ('radio' === updatedFormElement.elementConfig.type) {
```

Comment:
also this is one more "wtf" block - more readybility in terms of getting WHAT / WHAT-FOR it does - this way I can read it line by line in terms of HOW

Reply:
I can extract and put  into a arrow function to be more readable

Reply:
yes adopting clean-code principals and moving logic into named functional scopes will help a lot here :)